test/check-dashboard

```diff
@@ -188,9 +188,13 @@ class TestDashboard(MachineCase):
         m2.wait_boot()
 
         # Create some users on m1 and m2.
+        groups1 = m1.execute("groups")
+        if not "docker" in groups1:
```

Comment:
Where is this docker group coming from? Is this a regression in Atomic Host? I thought that Atomic Host shouldn't have a docker group by default.